Framework/API/src/DetectorInfo.cpp

```diff
@@ -580,5 +581,51 @@ boost::shared_ptr<const std::unordered_map<detid_t, size_t>>
 DetectorInfo::detIdToIndexMap() const {
   return m_detIDToIndex;
 }
+/**
+  @param  bankName     Name of bank containing peak
+  @param  col          Column number containing peak
+  @param  row          Row number containing peak
+  @param  Edge         Number of edge points for each bank
+  @return True if peak is on edge
+*/
+bool DetectorInfo::edgePixel(Mantid::Geometry::Instrument_const_sptr inst,
```

Comment:
As far as I can tell this static method makes no use of `API::DetectorInfo`. Would this be best as a free function in some other namespace?